apps/website/src/pages/index.astro

```diff
@@ -311,10 +321,18 @@ const currentPage = new URL(Astro.request.url).pathname;
     }
     .library-item {
       padding: var(--space-5);
-      text-align: start;
-      border: 1px solid var(--color-text);
       border-radius: var(--border-radius-1);
+      box-shadow: 0 0 1px var(--color-highlight-1), 0 0 10px var(--color-highlight-2);
```

Comment:
Are these changes intentional? Previously it only had box-shadow on hover/focus and had better visibility. It looks too dim/blue now.

| Before | After |
| --- | --- |
| ![](https://user-images.githubusercontent.com/9084735/207689066-df9b575f-3dfb-4744-8aec-fae4cd2c62b8.png) | ![](https://user-images.githubusercontent.com/9084735/207689299-9f94959b-fa40-4ef0-84bc-d2096afa2e88.png) |

Reply:
I wanted it to match copy to clipboard button but I undid all the changes because of better visibility.